apps/heureka/src/components/Services/common/ServiceImageVersions/ServiceImageVersions.tsx

```diff
@@ -55,14 +73,42 @@ export const ServiceImageVersions = ({ service, showDetailsButtons }: ServiceIma
     imageName: version.ccrn,
     imageVersion: version.version,
     imageTag: version.tag,
+    imageRepository: version.repository,
     issueCounts: version.issueCounts,
     serviceName,
+    componentInstances: version.componentInstances
+      ? {
+          totalCount: version.componentInstances.totalCount,
+          edges: version.componentInstances.edges,
+        }
+      : undefined,
   }))
 
-  const columnCount = showDetailsButtons ? 7 : 6
+  const columnCount = 7
+
+  const selectImageVersion = useCallback(
+    ({ service, imageVersion }: { service: ServiceType; imageVersion: ServiceImageVersion }) => {
+      dispatch({
+        type: ActionType.SelectImageVersion,
+        payload: {
+          service,
+          imageVersion,
+          showPanel: true,
+        },
+      })
+    },
+    []
+  )
 
   const showServiceDetails = useCallback(
```

Comment:
Not sure that all of this usecallbacks are needed. But If using it we need to add the dependencies like `dispatch `...